Docs/configure-preconfigured-demo.md

```diff
@@ -61,7 +61,7 @@ This article walked through changing the footprint of the underlying Azure servi
 
 
 <!-- Images and links -->
-[Microsoft Azure IoT Suite]: http://www.microsoft.com/server-cloud/internet-of-things/azure-iot-suite.aspx
+[Microsoft Azure IoT Suite]: https://www.microsoft.com/en-us/internet-of-things/azure-iot-suite
```

Comment:
My understanding from the email thread is that this should be:
 - https://www.microsoft.com/internet-of-things/azure-iot-suite 
That way the automatic language-locale redirection works.